### PR TITLE
feat: add hashtagsMatching method

### PR DESCRIPTION
**Pull request by achou11**
See https://gitlab.com/staltz/manyverse/-/issues/2169 for additional context

### CODE REVIEW

**Comment by achou11** [comment]
CI seems to be failing for an unrelated reason

**Comment by staltz** [comment]
Oh, could you drop Node 12.x and 14.x from CI?

**Comment by achou11** [comment]
> Oh, could you drop Node 12.x and 14.x from CI?

done. added 18 if that's okay

File: readme.md [hunk 1]
```diff
@@ -112,6 +112,12 @@ Returns a ("live") pull stream that emits the message key (string) for thread ro
 * `opts.allowlist`: optional array of strings. Dictates which messages **types** to allow as root messages, while forbidding other types.
 * `opts.blocklist`: optional array of strings. Dictates which messages **types** to forbid as root messages, while allowing other types.
 
+### `ssb.threads.hashtagsMatching(opts)`
```
Comment (staltz) [inline readme.md:115]: The API seems to be `(opts, cb)`, so this needs to change. Also the sentence says "returns a pull stream", should say "calls the callback with...". :)

**Review by staltz — CHANGES_REQUESTED** [review]
Implementation and tests look good, just have a tiny readme suggestion. :)

File: readme.md (hunk 1 above)
Comment (achou11) [reply readme.md]: addressed via 43d2270

**Review by staltz — APPROVED** [review]

**Comment by staltz** [comment]
Thank you!

**Pull request merged**